src/app/components/home-page-container/home-page-container.component.html

```diff
@@ -0,0 +1,102 @@
+<div fullpage id="fullpage" [options]="config" (ref)="getRef($event)">
+    <div class="section">
+        <div class="home-page-container shadow">
+            <div class="home-page-title">
+                <div class="header">Cockatoos</div>
+                <div class="subheader">Your 24/7 Speech Coach.</div>
+                <a class="button" mat-button [routerLink]="'/app'"> Get Started </a>
+            </div>
+            <img src="../../../assets/cockatoos.png" id="main-image" />
+        </div>
+        <div class="navigation">
+            <img src="../../../assets/arrow-down-sign-to-navigate.svg" (click)="fullpageApi.moveSectionDown()" />
```

Comment:
@neshang 
Note: I added a click event here so clicking on the down arrow will move down - I know that the down arrow means to scroll down, but IMO it should also work if someone clicks on it.